src/material/menu/menu-trigger.ts

```diff
@@ -76,6 +76,7 @@ const passiveEventListenerOptions = normalizePassiveListenerOptions({passive: tr
     'class': 'mat-menu-trigger',
     'aria-haspopup': 'true',
     '[attr.aria-expanded]': 'menuOpen || null',
+    '[attr.aria-owns]': 'menuOpen ? menu.panelId : null',
```

Comment:
I don't think `aria-owns` is appropriate here; none of the menu aria blueprints use it. My understanding is that `aria-owns` is more for subordinate relationships (like combobox -> listbox), rather than button -> menu, which are more separate. We can just add a `data-` attribute like `data-mat-menu-panel`

Reply:
Interesting, I'm surprised there's no standard aria attribute to link a trigger and popup together. It seems like something the accessibility model would want to know about. I'll change it though

Reply:
Switched to `aria-controls` which appears to be the correct attribute: https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html